src/main/java/uk/gov/hmcts/ccd/endpoint/std/CaseDetailsEndpoint.java

```diff
@@ -385,6 +398,7 @@ public CaseDetails createCaseEventForCitizen(
     @ApiResponses(value = {
         @ApiResponse(code = 200, message = "Documents list for the given case id")
     })
+    @LogAudit(operationType = OperationType.SEARCH_CASE)
     public List<Document> getDocumentsForCase(
```

Comment:
don't think it's correct to classify this as a search. There is not requirement to log this in the design. We could ignore or if we want to log it then we need  a more suitable operation name, like CASE_DOCUMENTS_ACCESS?